pkg/bff/organization.go

```diff
@@ -47,3 +49,45 @@ func (s *Server) OrganizationFromClaims(c *gin.Context) (org *models.Organizatio
 
 	return org, nil
 }
+
+// AddOrganizationCollaborator adds a new collaborator record to an organization.
```

Comment:
Let's add a note to the docstring that it's the caller's responsibility to save the organization back to the database.